### PR TITLE
Remove unused code and update app's description

### PR DESCRIPTION
**Pull request by raquelalegre**
We don't need any React code in this repository given the deployment occurs automatically from the latest [development repo](https://github.com/UCL/UKCORDEX-plot-explorer-dev) version. It can be confusing to have here a version of the code that doesn't correspond with what's been deployed in the production URL.

### CODE REVIEW

**Review by giordano — APPROVED** [review]
Changes here look good to me!  My only comment is that maybe all the information about how to develop the app should be moved to the README of the [development repository](https://github.com/UCL/UKCORDEX-plot-explorer-dev) (and maybe add a reference here)?  I don't see such information in the README there.

**Comment by raquelalegre** [comment]
Thanks! Yep, an updated version of what the development instructions here will be added in a new PR in the dev repo.

**Comment by raquelalegre** [comment]
Ah, forgot to say this was done in https://github.com/UCL/UKCORDEX-plot-explorer-dev/pull/63. Merging!

File: README.md [hunk 1]
```diff
@@ -1,72 +1,14 @@
...
-1. Create a `deploy` version of your `feature` or `bugfix` branch with the same content.
-1. Run
-    ```
-    npm build 
-    ```
-1. This would have created a build directory that contains an `index.html` file, amongst other things.
-1. Go to the GitHub repository and find "Settings" -> "Pages".
-1. Choose to publish the new `deploy` branch you have just created.
-1. Might need to wait a few minutes for the changes to propagate.
+The code in this repository has been moved to [the Plot Explorer's development repository](https://github-pages.ucl.ac.uk/UKCORDEX-plot-explorer-dev/#/how-to-use).
```
Comment (giordano) [inline README.md:13]: It just occurred to me this links to the development website, not the GitHub repository itself, as the name of the link would suggest.  Was it intentional?  To add to confusion, the linked page doesn't seem to have a link to the GitHub repository.

Comment (raquelalegre) [reply README.md:13]: Good catch! Thanks @giordano!

**Comment by raquelalegre** [comment]
Ok... I'll do it in another branch...

**Pull request merged**